### PR TITLE
add chapter 8

### PR DESCRIPTION
**Pull request by Ruslan357**

### CODE REVIEW

**Comment by Ruslan357** [comment]
Chapter 8

File: Chapter 8/8-4.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def make_shirt(size, text):
```
Comment (auraz) [inline Chapter 8/8-4.py]: Default value required according to the task

File: Chapter 8/8-5.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+def describe_city(name,country='Ukraine'):
+    '''format and show text'''
+    print(f'{name.title()} is a {country.title()}')
+
+
+describe_city('kyiv')
+describe_city('london')
```
Comment (auraz) [inline Chapter 8/8-5.py]: London is not in Ukraine

File: Chapter 8/8-11.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+def send_massages(list_of_text):
+    '''shows messages from the list. Creates and returns new list'''
+    sent_massages = []
+    while list_of_text:
+        temp = list_of_text.pop()
+        print(temp)
+        sent_massages.append(temp)
+    print()
```
Comment (auraz) [inline Chapter 8/8-11.py]: list is not returned

**Review by auraz — CHANGES_REQUESTED** [review]
Good job. 
Where is 8-15 - 8-17?

File: Chapter 8/8-4.py (hunk 1 above)
Comment (Ruslan357) [reply Chapter 8/8-4.py]: thanks, fixed it!

File: Chapter 8/8-5.py (hunk 2 above)
Comment (Ruslan357) [reply Chapter 8/8-5.py]: thanks, fixed it

File: Chapter 8/8-11.py (hunk 3 above)
Comment (Ruslan357) [reply Chapter 8/8-11.py]: thanks, fixed it

File: Chapter 8/.idea/.gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (auraz) [inline Chapter 8/.idea/.gitignore]: You've removed .idea from git, right?
.gitignore should live in the root folder of the repo.

**Review by auraz — CHANGES_REQUESTED** [review]
Add __pycache__ to gitignore

**Review by auraz — CHANGES_REQUESTED** [review]
Code looks good, but .idea, and .gitignore should be fixed.

File: Chapter 8/.idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (auraz) [inline Chapter 8/.idea/vcs.xml]: .idea folder should not be included in PR. These are your personal settings, not tasks from the book

**Review by auraz — CHANGES_REQUESTED** [review]

File: Chapter 8/.idea/.gitignore (hunk 4 above)
Comment (Ruslan357) [reply Chapter 8/.idea/.gitignore:1]: Я випустив ідею з git. Файл .gitignore в корневій папці репозиторія

Comment (Ruslan357) [reply Chapter 8/.idea/.gitignore]: I deleted .idea from git. File .gitignore in the root folder of the repository

File: Chapter 8/.idea/vcs.xml (hunk 5 above)
Comment (Ruslan357) [reply Chapter 8/.idea/vcs.xml]: Done! Removed from the repository

**Review by auraz — APPROVED** [review]
Good job.

**Pull request merged**